_sass/components/_identifier.scss

```
@@ -0,0 +1,140 @@
+.contact-bar {
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+}
+
+.usa-identifier {
+  background-color: $theme-identifier-background-color;
+  color: $theme-identifier-color;
+  padding-top: 1rem;
+  padding-bottom: 3rem;
+
+  font-family: $font-sans;
+  font-weight: 400;
+  font-size: 1.5rem;
+
+  a {
+    color: $theme-identifier-color;
+  }
+
+  ul, ol {
+    padding: 0;
+    margin: 0;
+    list-style: none;
+  }
+}
+
+.usa-identifier__required-links-list a {
+  font-size: 1.5rem;
+}
+
+a.usa-identifier__required-link {
```

The class should be enough and helps us keep selector specificity low for easier overrides in the future.

```suggestion
.usa-identifier__required-link {
```

---

element selector is removed